<ui version="4.0" >
 <class>AddServiceProcessWidget</class>
 <widget class="QWidget" name="AddServiceProcessWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>691</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Please enter a capacity for the service process:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLineEdit" name="expectationValueEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>10.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2" >
      <widget class="QDoubleSpinBox" name="factorSpin" >
       <property name="minimumSize" >
        <size>
         <width>95</width>
         <height>0</height>
        </size>
       </property>
       <property name="decimals" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" >
      <widget class="QLineEdit" name="standardDeviationEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>10.0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7" >
      <widget class="QDoubleSpinBox" name="capacitySpin" >
       <property name="decimals" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">(Expectation Value)&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">(Standard Deviation)&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item row="1" column="7" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">(Service Process Capacity)&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>Add Service Process</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>691</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
